src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin-right: 4px;
+    background-repeat: no-repeat;
+    block-size: cover;
+    background-position: center;
+    background-image: url("../../src/images/star.svg");
+
+    list-style: none;
+  }
+
+  &--4 &__star:nth-child(-n+4) {
+    background-image: url("../../src/images/star-active.svg");
+  }
```

Comment:
Why you didn't fix this 😢 
It will not work for other modifiers
For example I want to see only 3 active stars

<img width="967" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/86789161/9cba2e3d-a0ac-4c1c-8c57-90d57781ffaf">

Please, read task description more attentively

<img width="823" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/86789161/f92aa7a8-25fc-4cf8-afc8-b159b50b653f">